src/core-interfaces.ts

```diff
@@ -128,6 +128,11 @@ export interface HyperlinkProps {
 	 */
 	tooltip?: string
 }
+// used by: image, ...
```

Comment:
during dev I left the `...` here as a placeholder since I _think_ we might add tags to text as well but we haven't really built the fn to support that in the addin yet. Theoretically if we are correctly extending the library tags should be able to be applied to most, if not all shape types.

Reply:
ok sounds good to me